server/index.js

```diff
@@ -45,6 +18,10 @@ const app = express();
 var port = normalizePort(process.env.PORT || '3000');
 app.set('port', port);
 
+//You can set the environment variable DEVMODE to either false or true. It defaults to false if its not set.
+//Initialize the DEVMODE with the bash command: DEVMODE=true node index.js 
+var isDevelopmentMode = process.env.DEVMODE || false;
```

Comment:
I would suggest making it true by default. We are in developing mode unless we specifically set up production configuration.